src/components/handlers/Model3D.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+	<vue3dLoader
+	:showFps="false"
+	:cameraPosition="{ x: 1, y: -5, z: -100 }"
+	:filePath="src"
+	:lights="lights"
+	:crossOrigin="anonymous"
+	:backgroundAlpha="0"
+	class="img"
+	></vue3dLoader>
+</template>
+
+<script lang="ts" setup>
+import { vue3dLoader } from "vue-3d-loader";
+
+const props = defineProps(['src'])
+const lights = [
+  {
+    type: "AmbientLight",
+    color: "black",
+  },
+  {
+    type: "DirectionalLight",
+    position: { x: 100, y: 10, z: 100 },
+    color: "black",
+    intensity: 0.5,
+  },
+  {
+    type: "PointLight",
+    color: "#000000",
+    position: { x: 200, y: -200, z: 100 },
+    intensity: 1
+  },
+  {
+    type: "HemisphereLight",
+    skyColor: "#00FF00",
+    groundColor: "#000000",
+    position: { x: 200, y: -200, z: 100 }
+  }
+]
+</script>
+
+
+
+<style scoped>
```

Comment:
some of the styling here may be cleaned up later on, I used the Img handler as a base to create this one, so I brought in the styling as well